src/components/users/UserMembershipsList.tsx

```diff
@@ -0,0 +1,43 @@
+import { Group, User } from 'prisma/prisma-client';
+import styled from 'styled-components';
+import { gapM, gapS } from '@taskany/colors';
+
+import { NarrowSection } from '../NarrowSection';
+import { UserMemberships } from '../../modules/user.types';
+import { MembershipGroupListItemEditable } from '../MembershipGroupListItemEditable';
+import { InlineGroupSelectForm } from '../InlineGroupSelectForm';
+import { useUserMutations } from '../../modules/user.hooks';
+
+import { tr } from './users.i18n';
+
+const StyledMembershipList = styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: ${gapS};
+    margin-bottom: ${gapM};
+    width: 600px;
+`;
+
+type UserMembershipsProps = {
+    user: User & UserMemberships;
+};
+
+export const UserMembershipsList = ({ user }: UserMembershipsProps) => {
+    const { addUserToGroup } = useUserMutations();
+
+    const onAddUserToTeam = async (group: Group) => {
```

Comment:
When we use `Team` and when we use `Group`

Reply:
I try to use `Group` in code and `Team` in UI. There are some inconsistencies, so I sneak little refactorings into PR's :)